### PR TITLE
add monot5 tpu train doc

### PR DESCRIPTION
**Pull request by MXueguang**

### CODE REVIEW

File: pygaggle/data/create_msmarco_t5_training_pairs.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""
+This script creates monoT5 input files for training,
+Each line in the monoT5 input file follows the format:
+    f'Query: {query} Document: {document} Relevant:\t {label}\n')
```
Comment (rodrigonogueira4) [inline pygaggle/data/create_msmarco_t5_training_pairs.py:4]:
```suggestion
    f'Query: {query} Document: {document} Relevant:\t{label}\n')
```

File: docs/experiments-monot5-tpu.md [hunk 2]
```diff
@@ -211,4 +211,61 @@ You should see the same result.
...
+
+Copy pre-trained checkpoint to our target model
+```
+echo "model_checkpoint_path: \"model.ckpt-999900\"" > checkpoint
+gsutil cp checkpoint ${GS_FOLDER}
+gsutil cp gs://t5-data/pretrained_models/${MODEL}/model.ckpt-999900* ${GS_FOLDER}
+```
+
+```
+export EXPNO=002
```
Comment (rodrigonogueira4) [inline docs/experiments-monot5-tpu.md:249]: We don't need this, right?

File: docs/experiments-monot5-tpu.md [hunk 3]
```diff
@@ -211,4 +211,61 @@ You should see the same result.
...
+  --gin_param="utils.run.train_dataset_fn = @t5.models.mesh_transformer.tsv_dataset_fn" \
+  --gin_param="tsv_dataset_fn.filename = '${GS_FOLDER}/query_doc_pairs.train.tsv'" \
+  --gin_file="learning_rate_schedules/constant_0_001.gin" \
+  --gin_param="run.train_steps = 1100000" \
+  --gin_param="tokens_per_batch = 65536" \
+  >> out.log_exp${EXPNO} 2>&1 &
+
+tail -100f out.log_exp${EXPNO}
+```
+
```
Comment (rodrigonogueira4) [inline docs/experiments-monot5-tpu.md:270]: Maybe add a sentence describing how long does it take to train?

**Review by rodrigonogueira4 — COMMENTED** [review]
This looks great! Just a few minor comments

File: docs/experiments-monot5-tpu.md (hunk 2 above)
Comment (MXueguang) [reply docs/experiments-monot5-tpu.md]: @rodrigonogueira4  is base ckpt=999900 for all models in training? I made it configurable rn, if it is fixed, i'll change it back to hardcode

**Comment by MXueguang** [comment]
When this PR finish, https://github.com/castorini/pygaggle/issues/107 will be my next PR, fyi

**Review by ronakice — APPROVED** [review]
LGTM, merging

**Pull request merged**